projects/igniteui-angular/package.json

```diff
@@ -63,6 +63,7 @@
   "peerDependencies": {
     "@angular/common": "^7.0.3",
     "@angular/core": "^7.0.3",
+    "@angular/cli": "~7.0.5",
```

Comment:
Why would the Angular CLI be a peer dep?

Reply:
Nvm, saw you use it to check versions. Still a nope, though.
You can still infer the CLI version is outdated if core & common are.